backend/src/api/model/event.rs

```diff
@@ -519,6 +519,50 @@ impl AuthorizedEvent {
         Ok(event)
     }
 
+    pub(crate) async fn create_placeholder(event: NewEvent, context: &Context) -> ApiResult<Self> {
```

Comment:
This is missing authorization. Only people with the upload role should be able to call this.